releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -3,7 +3,30 @@ Azure App Configuration Push extension for Azure DevOps pipeline can be installe
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
 ### v5.0.0 - February, 02 2023
-Updated the task to require Node.js 16. It previously required 10
+* **Breaking change**:
+  With the update to the task to require Node.js 16. It previously required 10. The minimum agent version for running the task on self hosted agent changed.
+
+    **Before**:
+      [Azure Pipeline agent version 2.144.0](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.144.0) or later.
+
+    **After**:
+      [Azure Pipeline agent version 2.206.1](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.206.1) or later.
+
+* **Breaking change**:

```

Comment:
I'd recommend having a single breaking changes section with bullet points below of the breaking changes.

## Breaking Changes

* Updated the task to require Node.js 16. It previously required 10.
* Updated the minimum supported agent version to 2.206.1. Previously it was 2.144.0

Reply:
Updated to have a single breaking changes section with bullet points. 
To confirm for the Pull task since we only document one breaking change ie on the Node js upgrade, we don't need to have it in bullet points and can have the before and after sections?